app/src/main/java/edu/uc/kovaciad/slicetracker/services/MaterialService.kt

```diff
@@ -2,17 +2,17 @@ package edu.uc.kovaciad.slicetracker.services
 
 import android.app.Application
 import androidx.room.Room
-import edu.uc.kovaciad.slicetracker.dao.IBrandDAO
+import edu.uc.kovaciad.slicetracker.dao.IMaterialDAO
```

Comment:
Incorrect import directive made IMaterialDAO (A required import) inaccessible, resulting in MaterialService.kt to not function. See import directives here: https://kotlinlang.org/docs/packages.html?q=im#imports